store/src/java/com/zimbra/cs/service/account/ClientInfo.java

```diff
@@ -52,12 +56,36 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
             ToXML.encodeAttr(parent, Provisioning.A_zimbraSkinSecondaryColor, domain.getSkinSecondaryColor());
             ToXML.encodeAttr(parent, Provisioning.A_zimbraSkinSelectionColor, domain.getSkinSelectionColor());
             ToXML.encodeAttr(parent, Provisioning.A_zimbraWebClientLoginURL, domain.getWebClientLoginURL());
-            ToXML.encodeAttr(parent, Provisioning.A_zimbraWebClientLogoutURL, domain.getWebClientLogoutURL());
+            ToXML.encodeAttr(parent, Provisioning.A_zimbraWebClientLogoutURL, webClientLogoutURL);
             ToXML.encodeAttr(parent, Provisioning.A_zimbraWebClientStaySignedInDisabled, String.valueOf(domain.isWebClientStaySignedInDisabled()));
+            // TODO: ZCS-11319 update this line to read from LDAP property once this is moved out of LC.
+            // e.g. change the `split(LC.web_client_logoff_..)` -> `domain.getWebClientLogoffURLs()`
+            encodeAttrSkipLogoff(parent, webClientLogoutURL, StringUtils.split(LC.zimbra_web_client_logoff_urls.value()));
         }
         return parent;
     }
 
+    /**
+     * Add zimbraWebClientSkipLogoff to instruct the webclient to skip full logoff
+     * when sending EndSession request, if the configured zimbraWebClientLogoutURL is
+     * known to handle token de-registration.
+     *
+     * @param parent The element to add the attr to
+     * @param webClientLogoutURL The configured webclient logout url
+     * @param logoffURLs URLs to skip logoff for - we expect them to handle it when we send the user there
+     */
+    protected void encodeAttrSkipLogoff(Element parent, String webClientLogoutURL, String[] logoffURLs) {
+        // always include the attr if webclient logout url is non-empty
+        if (StringUtils.isNotEmpty(webClientLogoutURL)) {
```

Comment:
We have one more method to check both bull and empty on String in com.zimbra.common.util.StringUtil class

Reply:
I think we should update to null & empty check, as the default value for attribute is null if not set

Reply:
StringUtils.isNotEmpty does the same, so its fine.

Reply:
OK